CHANGELOG.md

```diff
@@ -7,6 +7,24 @@
 * Breaking: dropped Svelte 2 support ([#150](https://github.com/sveltejs/svelte-loader/pull/150))
 * Breaking: dropped Node 8 support ([#157](https://github.com/sveltejs/svelte-loader/pull/157))
 * Add Webpack 5 support ([#151](https://github.com/sveltejs/svelte-loader/pull/151))
```

Comment:
Can we move this bullet to be the last since it's got a long sub-bullet under it now? I think that'd make it slightly easier to read

Reply:
Removed the code snippet so not needed anymore